src/rdkafka_admin.c

```diff
@@ -7131,60 +7209,67 @@ const rd_kafka_group_result_t **rd_kafka_ListConsumerGroupOffsets_result_groups(
 static rd_kafka_ConsumerGroupListing_t *
 rd_kafka_ConsumerGroupListing_new(const char *group_id,
                                   rd_bool_t is_simple_consumer_group,
-                                  rd_kafka_consumer_group_state_t state) {
-        rd_kafka_ConsumerGroupListing_t *grplist;
-        grplist                           = rd_calloc(1, sizeof(*grplist));
-        grplist->group_id                 = rd_strdup(group_id);
-        grplist->is_simple_consumer_group = is_simple_consumer_group;
-        grplist->state                    = state;
-        return grplist;
+                                  rd_kafka_consumer_group_state_t state,
+                                  rd_kafka_consumer_group_type_t group_type) {
+        rd_kafka_ConsumerGroupListing_t *group;
+        group                           = rd_calloc(1, sizeof(*group));
```

Comment:
`consumer_group_listing` or `cg_listing` better than `group`. Same for other new APIs as well.